stellar/frontend/src/app/components/pages/application/create-apply-project/create-project-form/CreateProjectStep5.tsx

```
@@ -312,7 +374,10 @@ const CreateProjectStep5 = () => {
 							</div>
 						)}
 						{embededYtHtml && (
-							<div dangerouslySetInnerHTML={{ __html: embededYtHtml }} />
+							<div
+								className="overflow-hidden rounded-b-xl"
+								dangerouslySetInnerHTML={{ __html: embededYtHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)